.github/workflows/releases.yml

```
@@ -1,54 +1,57 @@
           echo "$RELEASE_NOTES" > RELEASE_NOTES.md
-
+          
           # Pass release notes to GitHub Actions output as multiline
           {
             echo "release_notes<<EOF"
             echo "$RELEASE_NOTES"
             echo "EOF"
           } >> "$GITHUB_OUTPUT"
-
+      
       - name: Create GitHub Release
```

actionlint (1.7.8)

step must run script with "run" section or run action with "uses" section

(syntax-check)